openmdao/utils/file_utils.py

```diff
@@ -481,3 +489,117 @@ def _get_outputs_dir(obj=None, *subdirs, mkdir=True):
         dirpath.mkdir(parents=True, exist_ok=True)
 
     return dirpath
+
+
+def _is_openmdao_output_dir(directory):
+    """
+    Check if a directory is an OpenMDAO output directory.
+
+    Parameters
+    ----------
+    directory : str or Path
+        The directory to check.
+
+    Returns
+    -------
+    bool
+        True if the directory is an OpenMDAO output directory, False otherwise.
+    """
+    directory = pathlib.Path(directory)
+    return directory.is_dir() and directory.name.endswith('_out') \
```

Comment:
it seems like the directory_name check is unnecessary? the presence of the dot file is conclusive...
(maybe the is_dir check is redundant as well)

Reply:
It probably is, but when it comes to automatically removing things from a user's system I wanted to do belt and suspenders.